www/src/pages/index.tsx

```diff
@@ -61,6 +61,41 @@ const useQueryParam = (key: string) => {
   return new URLSearchParams(location.search).get(key);
 };
 
+function ErrorOverlay(props: { url: string; dismiss: () => void }) {
+  return (
+    <div className="absolute inset-0 z-10 flex flex-col items-center justify-center gap-2 overflow-hidden bg-zinc-900">
+      <div className="flex flex-col items-center justify-center gap-4">
+        <div className="text-5xl font-bold text-zinc-900 dark:text-zinc-100">
+          ⚠️
+        </div>
+        <div className="text-xl font-bold text-zinc-900 dark:text-zinc-100">
+          Heads up!
+        </div>
+        <div className="w-full max-w-xl text-lg font-medium text-zinc-700 dark:text-zinc-200">
+          WebContainers require modern Web APIs that likely isn&apos;t supported
+          by your browser. Either open this page in a different browser, or
```

Comment:
Maybe just say Chrome / Chromium? Or is there any other browser/engine with good enough support?